plane/source/docs/guide-four-channel-plane.rst

```diff
@@ -26,7 +26,9 @@ After the RC inputs are configured, configure the outputs.
              
 Servo cables can be connected to any output of the autopilot,
 but using the default channels 1-4 listed below is recommended. 
-Set the SERVOn_FUNCTIONS to the appropriate values.
+Set the SERVOn_FUNCTION to the appropriate values.
+
+You can do this by setting the raw parameter, or using Ground Station software such as Mission Planner or QGroundControl. In Mission Planner you can do this on the Setup -> Servo Output page. Each channel has a dropdown which can be used to select the function of the channel. In QGroundControl this gets set when you use the Calibrate Radio option. If you use this it actually sets the SERVOn_FUNCTION under the covers
```

Comment:
the last sentence is not true...calibrating radio in any GCS does not impact the SERVOn_FUNCTION assignments

Reply:
the  menu item in MP is SETUP/Mandatory Hardware/Servo Output.... not SETUP/Servo Output

Reply:
Thanks for the feedback @Hwurzburg. Yes I understand what you are saying about the radio calibration. I'm setting up another plane next week, so I'll run through this again.

Reply:
> if you are doing this via the direct wiki editor, I can make these changes for you since you will not be able to do them....let me know...thanks

Sure - perhaps you can explain to me what I should have done, I did it this way because that's what I thought I needed to do. (click "Edit on GitHub" from the page itself).

Also - this page seems to be orphaned. There is a similar page that is in the doc hierarchy, but this one doesn't appear in the wiki menu and you can't easily find it by just navigating. Do you think we can fix that too?

Reply:
Those buttons are great for small changes! 

But once you need to edit the changes a bit you will end up with a bunch of extra commits. When we merge to master Henry will likely only want this be one commit. "Plane: update guide four channel plane"

Currently you have a 4 commits and one of those is merge commit which is no-no in our git workflow.

Usually, I'd point you to the wiki but our servers are down for a few more hours ..... so the links are the source code not the pretty wiki pages. 

Here is the page about how to squash and rebase commits: 

https://github.com/ArduPilot/ardupilot_wiki/blob/master/dev/source/docs/git-interactive-rebase.rst
https://github.com/ArduPilot/ardupilot_wiki/blob/master/dev/source/docs/git-rebase.rst

This is how to setup for building the wiki.  https://github.com/ArduPilot/ardupilot_wiki/blob/master/common/source/docs/common-wiki-editing-setup.rst

You may not need to do actually build the wiki.

Reply:
> Sure - perhaps you can explain to me what I should have done, I did it this way because that's what I thought I needed to do. (click "Edit on GitHub" from the page itself).
> 
> Also - this page seems to be orphaned. There is a similar page that is in the doc hierarchy, but this one doesn't appear in the wiki menu and you can't easily find it by just navigating. Do you think we can fix that too?
you are doing fine...you are using the built in editor in the wiki....it just wont let you do changes to your PR easily or "squash" the suggestions from others that you commit....that requires the full wiki editing environment, learning git, etc....dont worry about that unless you want to....I am happy to pull the PR down locally edit, rebase, and squash, which I will do in a minute....
The page is not orphaned....that cant happen in the wiki without creating a build error...its linked from the first time setup->configuration->choosing servo functions page

Reply:
Henry - try this - go to this page in the Wiki. The breadcrumb shows as Docs » Setup for Plane » Plane Configuration » Choosing Servo Functions » Four Channel Planes.
Then click on it's parent "Choosing Servo Functions" - now try to get back to the Four Channel Planes page using the left navigation menu - you can't!

Reply:
its a matter of what heading get shown in the left pane...TOC content pages are not headings within a document...this is normal for all pages that directly call other pages...the TOC links are not listed in the left menu....and would not want them there as they would clutter it up in many cases....its link is  under the Setup of Specific Aircraft Types section of that page